src/main/java/org/opentripplanner/routing/edgetype/ElevatorHopEdge.java

```diff
@@ -29,33 +31,74 @@ public ElevatorHopEdge(
     Vertex from,
     Vertex to,
     StreetTraversalPermission permission,
+    WheelchairAccessibility wheelchairAccessibility,
     double levels,
     int travelTime
   ) {
-    this(from, to, permission);
+    this(from, to, permission, wheelchairAccessibility);
     this.levels = levels;
     this.travelTime = travelTime;
   }
 
-  public ElevatorHopEdge(Vertex from, Vertex to, StreetTraversalPermission permission) {
+  public ElevatorHopEdge(
+    Vertex from,
+    Vertex to,
+    StreetTraversalPermission permission,
+    WheelchairAccessibility wheelchairAccessibility
+  ) {
     super(from, to);
     this.permission = permission;
+    this.wheelchairAccessibility = wheelchairAccessibility;
+  }
+
+  public static void bidirectional(
+    Vertex from,
+    Vertex to,
+    StreetTraversalPermission permission,
+    WheelchairAccessibility wheelchairBoarding,
+    int levels,
+    int travelTime
+  ) {
+    new ElevatorHopEdge(from, to, permission, wheelchairBoarding, levels, travelTime);
+    new ElevatorHopEdge(to, from, permission, wheelchairBoarding, levels, travelTime);
+  }
+
+  public static void bidirectional(
+    Vertex from,
+    Vertex to,
+    StreetTraversalPermission permission,
+    WheelchairAccessibility wheelchairBoarding
+  ) {
+    new ElevatorHopEdge(from, to, permission, wheelchairBoarding);
+    new ElevatorHopEdge(to, from, permission, wheelchairBoarding);
   }
 
   public StreetTraversalPermission getPermission() {
     return permission;
   }
 
+  @Override
   public String toString() {
-    return "ElevatorHopEdge(" + fromv + " -> " + tov + ")";
+    return ToStringBuilder.of(this.getClass()).addObj("from", fromv).addObj("to", tov).toString();
   }
 
   @Override
   public State traverse(State s0) {
-    RoutingRequest options = s0.getOptions();
+    RoutingRequest request = s0.getOptions();
```

Comment:
I've always wondered why the request is called options in the state.

Reply:
I think it's because a very long time ago RoutingRequest was called RoutingOptions.